### PR TITLE
Add com.zandronum.Zandronum

### PR DESCRIPTION
**Pull request by Eonfge**
Compare and contrast #1155 

Zandronum is the go to place for Multiplayer Doom. It's crazy moddable, it has a loyal fan base, and it's now also Flatpak supported.

It did require some puzzling because their build system relies on Mercurial, but I was able to make it all work in the end.

### CODE REVIEW

**Comment by Eonfge** [comment]
bot, build com.zandronum.Zandronum

**Comment by Eonfge** [comment]
bot, build com.zandronum.Zandronum

**Comment by LeandroStanger** [comment]
bot, build com.zandronum.Zandronum

File: com.zandronum.Zandronum.desktop [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Categories=Games;
+Comment=Leading the way in newschool multiplayer Doom online
+Categories=Game;Shooter;
```
Comment (bilelmoussaoui) [inline com.zandronum.Zandronum.desktop]: The categories are duplicated here.

Comment (Eonfge) [reply com.zandronum.Zandronum.desktop]: First one is also wrong I now see. Must not be plural.

**Comment by Eonfge** [comment]
bot, build com.zandronum.Zandronum

File: com.zandronum.Zandronum.yaml [hunk 2]
```diff
@@ -0,0 +1,161 @@
+app-id: com.zandronum.Zandronum
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: "5.12"
```
Comment (LeandroStanger) [inline com.zandronum.Zandronum.yaml]: Update runtime to 5.13

Comment (Eonfge) [reply com.zandronum.Zandronum.yaml]: Wasn't aware that 5.13 is available. First time I heard of it. Anyway, I updated my package.

**Comment by Eonfge** [comment]
bot, build com.zandronum.Zandronum

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.zandronum.Zandronum Read https://github.com/flathub/flathub/wiki/App-Maintenance for more on how Flathub works for app builders and accept the GitHub request you'll soon be sent!

**Pull request closed without merging**